entity-framework/toc.yml

```diff
@@ -423,7 +423,7 @@
         - name: Spatial data
           displayName: GIS
           href: core/providers/sqlite/spatial.md
-        - name: Microsoft.Data.Sqlite >>
+        - name: Microsoft.Data.Sqlite
```

Comment:
I think the `>>` is an indication that this directs users to a page outside the EF conceptual documentation, ilke other links below - do you think there's a reason to remove that? If so, I think we'd do it everywhere, not just for this one.